sign_language_datasets/datasets/chicagofswild/chicagofswild.py

```diff
@@ -126,4 +147,11 @@ def _generate_examples(self, archive_path: str, split: str):
 
                         datum["video"] = [path.join(frames_base, name) for name in sorted(tf.io.gfile.listdir(frames_base))]
 
+                    if self.builder_config.include_pose is not None:
+                        if self.builder_config.include_pose == "holistic":
+                            mediapipe_path = path.join(_POSE_URLS['holistic'], f"{_id}.pose")
+                            pose = Pose.read(open(mediapipe_path, "rb").read())
+                            pose = pose.get_components(["POSE_LANDMARKS", "FACE_LANDMARKS", "LEFT_HAND_LANDMARKS", "RIGHT_HAND_LANDMARKS"])
+                            datum["pose"] = pose
```

Comment:
Is this the desired way to load the pose?

Reply:
a few issues:
1. you are opening the file but not closing it. use `with open`
2. I don't think we should be making a judgement call on which components to `get`. use all of them in my opinion, and let the user decide. (changing the components here also makes the header change)

Reply:
Sure with 1. Regarding 2, this is defined previously in line 80 (543 points), as in some other datasets (signsuisse). I thought you wanted to remove the world landmarks by default. For sure we can make them included by first changing line 80. What do you think?

Reply:
I feel like there is no need to remove stuff, since someone might want to use them

Reply:
Okay let's keep the full 576 keypoints